src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -37,8 +49,20 @@ export const QuarterlyGraphs = props => {
             history: props.history,
           }}
         />
-        <SectionFilerInfo show={section === 'Filer Info'} />
-        <SectionFAQ show={section === 'FAQ'} />
+
+        <Switch>
+          {/* Setting direct paths to access other tabs */}
+          <Route
+            path={"/data-browser/graphs/quarterly/info/filers"}
+            render={() => (
+              <SectionFilerInfo show={section.label === "Filers' Info"} />
+            )}
+          />
+          <Route
+            path={"/data-browser/graphs/quarterly/info/faq"}
+            render={() => <SectionFAQ show={section.label === "FAQ"} />}
```

Comment:
With the above changes, we can remove the `show` prop since the `path` now determines which `<Section>` should be displayed.  
 
Inside of each `<Section>` we can also remove the conditional rendering using `if(!show)`.  And since that functionality was the main reason for splitting their code into Components, we can probably flatten the hierarchy and replace `SectionFilerInfo/SectionFAQ` with the content that they render.

Reply:
I flattened things out and deleted the SectionFilerInfo/SectionFAQ components.